src/migrations/20200129005352-create-company-profile.ts

```diff
@@ -0,0 +1,48 @@
+import { DATE, INTEGER, QueryInterface, STRING } from "sequelize";
+
+export = {
+    up: (queryInterface: QueryInterface) => {
+        return queryInterface.createTable("CompanyProfiles", {
+            id: {
```

Comment:
che y si usamos uuids? Se que capaz es un overkill pero seria lo correcto

Reply:
veo que hay soporte para eso, no estaria mal (espero que se autogenere igual... no me gustaria tener codigo que genera strings random tirado por todos lados jajaj)

https://stackoverflow.com/questions/49864723/sequelize-primarykey-as-uuid-in-mysql-database-not-working/49876720

Reply:
jajaja bien visto!

Reply:
Se patea para otro PR donde se haga una migracion que pase a UUID